src/frontend/Cargo.toml

```diff
@@ -29,12 +29,14 @@ parking_lot = "0.12"
 paste = "1"
 pgwire = { path = "../utils/pgwire" }
 rand = "0.8"
+risingwave_batch = { path = "../batch" }
 risingwave_common = { path = "../common" }
 risingwave_expr = { path = "../expr" }
 risingwave_pb = { path = "../prost" }
 risingwave_rpc_client = { path = "../rpc_client" }
 risingwave_source = { path = "../source" }
 risingwave_sqlparser = { path = "../sqlparser" }
+risingwave_storage = { path = "../storage" }
```

Comment:
Why frontend depends on storage and batch?

Reply:
Because the task context trait requires it.

Reply:
It depends on batch crate anyway, but we can do refactor to remove dependency to storage crate. Let's   merge this first, and I'll do refactor later to remove dependency to storage. cc @fuyufjh

Reply:
Sure, see #2743